src/main/java/com/beowulfe/hap/impl/characteristics/common/ObstructionDetectedCharacteristic.java

```diff
@@ -18,7 +18,7 @@ public ObstructionDetectedCharacteristic(
       Supplier<CompletableFuture<Boolean>> getter,
       Consumer<HomekitCharacteristicChangeCallback> subscriber,
       Runnable unsubscriber) {
-    super("00000024-0000-1000-8000-0026BB765291", false, true, "Obstruction Detected");
+    super("00000024-0000-1000-8000-0026BB765291", false, true, null);
```

Comment:
Do we really need to set these to these to null? Or should would modify the API responder to exclude the value?

Reply:
I can add an override so that we don't have to pass an explicit null, but I think it's useful if you're implementing a custom characteristic to be able to add a description. I don't think Home.app will show the custom characteristic at all, but I know there are 3rd party HomeKit apps that will show all characteristics. You know, a characteristic like https://github.com/timcharper/homebridge-vivint/blob/master/lib/thermostat_characteristics.js#L39 ;)